app/index.html

```diff
@@ -3,28 +3,54 @@
 <head>
     <meta charset="UTF-8">
     <title>Image Stitcher by Markus Zeller</title>
-    <link rel="stylesheet" href="css/style.css">
+    <link href="css/output.css" rel="stylesheet">
     <script src="js/main.js" defer></script>
 </head>
 <body>
-<div class="bar">
-    <div>
-        <input type="radio" name="mode" id="mode-horizontal" value="horizontal" checked>
-        <label for="mode-horizontal">Horizontal</label>
-    </div>
-    <div>
-        <input type="radio" name="mode" id="mode-vertical" value="vertical">
-        <label for="mode-vertical">Vertical</label>
+
+<!-- tailwinds layout for an app in a container -->
```

Comment:
Please, no comments in output

Reply:
That's true. Sloppy, sorry mate. Addressed in [8a3f7fa](https://github.com/markuszeller/image-stitcher/pull/3/commits/8a3f7fa202ec69bd443981835a8f8070f9b274d3)